assets/storageclass.yaml

```diff
@@ -0,0 +1,9 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: standard-csi
```

Comment:
I picked a random name here. Is there a better name?

Reply:
Looks like same name is being used by other CSI drivers too. So may be we should stick to it.

Reply:
The notation I used in other CSI SCs was: _default_sc_name_used_intree + "-csi"_.

`standard` is the default SC for GCE I think (where this was probably copied from). I think Cinder copied GCE too.

I'm fine with this, but I'd be nice to use the same name as the default SC for intree plugin and append `-csi`.

Reply:
hmm, that would logically lead to `thin-csi` . I think there are no better names as such here. I have no strong opinions on this one. Do you prefer `thin-csi` ? I could name it `thin-csi`.

Reply:
I renamed it to `thin-csi`